addons/hr_attendance/models/hr_attendance.py

```diff
@@ -16,36 +16,116 @@
 from odoo.exceptions import AccessError
 
 
+def float_time_to_string(float_time):
+    hours = int(float_time)
+    minutes = int((float_time - hours) * 60)
+    return f"{hours}h{minutes if minutes else ''}"
```

Comment:
This is not translated and localized according to the user's preferences.
Use from ``tools/misc.py``
```py
def format_duration(value):
    """ Format a float: used to display integral or fractional values as
        human-readable time spans (e.g. 1.5 as "01:30").
    """
```